### ISSUE TITLE
eslint v5 support

### ISSUE THREAD

**Issue by ljharb** [airbnb/javascript#1834]
Just like #1447, I'm filing this in advance.

v17.0.0 of `eslint-config-airbnb` and v13.0.0 of `eslint-config-airbnb-base` have just been released.

eslint v5 is planned to be released in the next 24-48 hours, and the plan for our configs will be to either support v3 || v4 || v5, or, if necessary, to drop support for v3 and/or v4 as part of this release.

The timeframe is "probably not as soon as you would like, but as soon as we can".
Please accept this, and please add no noise to this thread beyond emoji reactions - distractions won't help anyone get the work done any faster.

### **Please** do not file additional issues and do not file pull requests attempting to add support - please, instead, continue to use eslint v4 until such time as our configs support v5.

Peer Dep eslint combined v3 and v4 and v5 support released, or v5 support released (depending on which path we need to take):
 - [x] `eslint-plugin-import`
    - [x] [PR](https://github.com/benmosher/eslint-plugin-import/pull/1122)
    - [x] Released as v2.13.0
 - [x] `eslint-find-rules`
   - [x] [PR](https://github.com/sarbbottam/eslint-find-rules/pull/296)
   - [x] [Release](https://github.com/sarbbottam/eslint-find-rules/pull/300) as v3.3.1
 - [x] update the base config
 - [x] publish the base config (v13.1.0)
 - [x] `eslint-plugin-react`
   - [x] [PR](https://github.com/yannickcr/eslint-plugin-react/pull/1843)
   - [x] Released as v7.10.0
 - [x] `eslint-plugin-jsx-a11y`
   - [x] [PR](https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/451)
   - [x] [Released](https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/452) as v6.1.0
 - [x] eslint-plugin-react
   - [x] enable new option in (jsx-one-expression-per-line](https://github.com/yannickcr/eslint-plugin-react/issues/1775)
   - [x] released as v7.11.0
   - [x] updated in the main config
 - [x] a11y plugin: fix [idlist bug](https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/454) ([PR](https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/456)) and release
 - [x] a11y plugin: [label-has-for bug](https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/455)
 - [x] update and publish the main config

**Comment by ljharb** [comment]
I'm waiting on a few bugs (that i've just added).

**Comment by bjankord** [comment]
Hey @ljharb, curious what that status is on this. I saw some bugfixes landed in [jsx-a11y-plugin](https://github.com/evcohen/eslint-plugin-jsx-a11y/releases/tag/v6.1.1).

**Comment by danactive** [comment]
I would love to read why "react/jsx-one-expression-per-line" is enabled. Is there a trend for developers to rotate their monitors vertically after this change 😆

This changes one readable line
```
<App>Hello World</App>
```

to an uncomfortable syntax
```
<App>
  { 'Hello' }
  { ' ' }
  { 'World' }
</App>
```

**Comment by ryansully** [comment]
@danactive That rule was turned off in https://github.com/airbnb/javascript/commit/b6a268f780177e03b573a4f0df95ecc0d2e8783e.
Hopefully the next release that includes the commit above will be out soon.

Your concern was also brought up [here](https://github.com/yannickcr/eslint-plugin-react/issues/1848) and [here](https://github.com/yannickcr/eslint-plugin-react/issues/1855).

**Comment by Berkmann18** [comment]
Is there any plans on releasing a version which supports ESLint v5?

**Comment by olstenlarck** [comment]
berkman... that issue is exactly for that and they are working on it... be patient.

**Comment by stepankuzmin** [comment]
https://github.com/yannickcr/eslint-plugin-react/issues/1775 is closed, any plans on publishing new version?

**Comment by ljharb** [comment]
@stepankuzmin I’m not sure what answer you’re hoping to get? Yes, there are plans to publish a new version, at some point, when it’s ready and time permits.

**Comment by ljharb** [comment]
eslint-config-airbnb v17.1.0 and eslint-config-airbnb-base v13.1.0 have been released, with eslint 5 support.

**Comment by olstenlarck** [comment]
Yeah!

**Comment by kmaitski** [comment]
Is anyone still having the same issue after upgrading?

Here are my devDependencies, but I am still running into the same issue. 

`       "eslint": "^5.3.0",
        "eslint-config-airbnb": "^17.1.0",
        "eslint-config-airbnb-base": "^13.1.0",
        "eslint-loader": "^2.1.0",
        "eslint-plugin-import": "^2.14.0",
        "eslint-plugin-jsx-a11y": "^6.1.1",
        "eslint-plugin-react": "^7.10.0",
`

Am I doing something wrong?

**Comment by ljharb** [comment]
@kmaitski what does `npm ls` say? Have you tried running the peer deps install command in the readme for `eslint-config-airbnb`?

Either way, please file a new issue if you're still having trouble.

**Comment by ljharb** [comment]
@noahbenham can i ask why you're randomly thumbs down-ing comments? It seems pretty trollish and I'm hoping you can explain.
